docs/_content/standaard/standaard.md

```diff
@@ -141,6 +142,57 @@ foutbericht, waarbij de `invalid-params` key meer informatie bevat over de fout.
 Indien een parameter wordt toegepast die niet in de OAS van de betreffende API
 staat, dan MOET de API antwoorden met een HTTP 400 foutbericht.
 
+## Notificaties
+
+Componenten dienen events te publiceren naar (een) notificatiecomponent(en)
+(NC). De NC MOET volledig de
+[`openapi.yaml`](../../../api-specificatie/nc/openapi.yaml) implementeren.
+
+### Kanalen
+
+Elke bron, wat bij de ZGW API's één-op-éen overeen komt met een component
+zoals het ZRC, DRC, BRC, etc., MOETEN hun kanaal registreren bij de NC indien
+dit nog niet bestaat. Elke bron MOET tevens documenteren op welke kanalen die
```

Comment:
kanalen (meervoud)

Reply:
We gaan voor ZGW uit van 1 kanaal per bron